cloudfoundry/resource_cf_app_test.go

```diff
@@ -1321,6 +1353,78 @@ func TestAccApp_dockerApp(t *testing.T) {
 		})
 }
 
+func TestAccApp_url_docker_switch(t *testing.T) {
+
+	_, orgName := defaultTestOrg(t)
+	spaceID, spaceName := defaultTestSpace(t)
+	refApp := "cloudfoundry_app.java-spring"
+
+	resource.Test(t,
+		resource.TestCase{
+			PreCheck:     func() { testAccPreCheck(t) },
+			Providers:    testAccProviders,
+			CheckDestroy: testAccCheckAppDestroyed([]string{"java-spring"}),
+			Steps: []resource.TestStep{
+
+				resource.TestStep{
+					Config: fmt.Sprintf(fmt.Sprintf(appResourceUrlDockerTemplate, defaultAppDomain(), orgName, spaceName),
+						`docker_image = "cloudfoundry/diego-docker-app:latest"`,
+					),
+					Check: resource.ComposeTestCheckFunc(
+						testAccCheckAppExists(refApp, func() (err error) {
+
+							if err = assertHTTPResponse("https://java-spring."+defaultAppDomain(), 200, nil); err != nil {
+								return err
+							}
+							return
+						}),
+						resource.TestCheckResourceAttr(refApp, "name", "java-spring"),
+						resource.TestCheckResourceAttr(refApp, "space", spaceID),
+						resource.TestCheckResourceAttr(refApp, "ports.#", "1"),
+						resource.TestCheckResourceAttr(refApp, "ports.8080", "8080"),
+						resource.TestCheckResourceAttr(refApp, "instances", "1"),
+						resource.TestCheckResourceAttr(refApp, "memory", "768"),
+						resource.TestCheckResourceAttr(refApp, "disk_quota", "512"),
+						resource.TestCheckResourceAttrSet(refApp, "stack"),
+						resource.TestCheckResourceAttr(refApp, "environment.%", "0"),
+						resource.TestCheckResourceAttr(refApp, "enable_ssh", "true"),
+						resource.TestCheckResourceAttr(refApp, "health_check_type", "port"),
+						resource.TestCheckNoResourceAttr(refApp, "route.#"),
+						resource.TestCheckResourceAttr(refApp, "routes.#", "1"),
+					),
+				},
+
+				resource.TestStep{
+					Config: fmt.Sprintf(fmt.Sprintf(appResourceUrlDockerTemplate, defaultAppDomain()),
+						`url = "file://../tests/cf-acceptance-tests/assets/java-spring/java-spring.jar"`,
```

Comment:
Would it make sense to try to speed up tests by using static buildpack app rather than java buildpack (which is slower to stage and start) ? 

E.g. potentially reuse https://github.com/cloudfoundry/cf-acceptance-tests/tree/master/assets/staticfile